network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/featured_blogs.html

```diff
@@ -18,48 +18,60 @@ <h4 class="h4-heading">{% trans "Latest news" %}</h4>
     <div class="row">
       <div class="col-12 full-bleed-xs">
         <div class="key-item mx-2 mx-md-4 p-3">
-          <p class="h6-heading mb-2">
-            {% for category in blog.blog.specific.category.all %}
-            <a href="/blog/category/{{category.slug}}">{{ category }}</a>{% if not forloop.last %}, {% endif %}
-            {% endfor %}
-          </p>
           <h5 class="mb-2 h3-heading">
             <a href="{{blog.blog.url}}">{{blog.blog.title}}</a>
           </h5>
-          <p class="body-small mb-2">
-            {% if blog.blog.author %}
-              {% blocktrans with author=blog.blog.author %}by {{author}}{% endblocktrans %}
-            {% endif %}
-            {% if blog.blog.first_published_at %} {% blocktrans with date=blog.blog.first_published_at|date:"DATE_FORMAT" %}on {{date}}{% endblocktrans %}{% else %}not published yet{% endif %}
-          </p>
-          <p>{{blog.blog.search_description}}</p>
         </div>
       </div>
     </div>
   </div>
   {% endif %}
   {% endfor %}
 
+
   {% if featured_blogs|length > 1 %}
   <div class="col-md-6 mb-0 pb-4">
     {% for blog in featured_blogs %}
+    {% get_authors blog as authors %}
+
     {% if forloop.counter > 1 %}
     <div>
-      <p class="h6-heading mb-2">
-        {% for category in blog.blog.specific.category.all %}
-        <a href="/blog/category/{{category.slug}}">{{ category }}</a>{% if not forloop.last %}, {% endif %}
-        {% endfor %}
-      </p>
       <h5 class="mb-2">
         <a href="{{blog.blog.url}}">{{blog.blog.title}}</a>
       </h5>
-      <p class="body-small mb-2">
-        {% if blog.blog.author %}
-          {% blocktrans with author=blog.blog.author %}by {{author}}{% endblocktrans %}
-        {% endif %}
-        {% if blog.blog.first_published_at %} {% blocktrans with date=blog.blog.first_published_at|date:"DATE_FORMAT" %}on {{date}}{% endblocktrans %}{% else %}not published yet{% endif %}
-      </p>
-      <p>{{blog.blog.search_description}}</p>
+      <div class="body-small mb-2">
+
+{# TODO: Export this internal CSS to .scss #}
+<style>
+  .featured-blog-author-image {
+    display: inline-block;
+    width: 50px;
+    height: 50px;
+    position: relative;
```

Comment:
Just a note for whoever reviews this, this isn't "proper" css, it's just stuff I whipped up to test lining up images together.